README.md

```diff
@@ -154,6 +167,69 @@ To build your stdlib run:
 stdlib-builder /path-to-your/std-lib -po ./stdlib.json
 ```
 
+
+### Configuration actual for both
+
+#### Positional arguments:
+
+__DVM__ and __compiler__ requires second positional argument described as `<data-source address>`.
+This is URI of a data source server, typically [Dnode][], local or external.
+This argument can be ommited because we'll read the `DVM_DATA_SOURCE` [environment variable][environment variables] as fallback.
+
+Positional arguments have higher priority than [environment variables][], so overrides their if specified.
```

Comment:
meh.

, and override them when specified.

Reply:
Same fix below.

Reply:
fixed

Reply:
fixed